fem/bilininteg.hpp

```diff
@@ -75,6 +75,22 @@ class BilinearFormIntegrator : public NonlinearFormIntegrator
        called. */
    virtual void AddMultTransposePA(const Vector &x, Vector &y) const;
 
+   /// Method defining element assembly.
```

Comment:
This class starts to hold a lot of variants of itself, couldn't we try to split it like we did for `BilinearForm`?
We should discuss it.

Reply:
Would that mean using multiple inheritance when we implement PA or EA for an `Integrator`? I could see some design benefits to this, e.g. Volume vs Face Integrators.

Reply:
Yes, like `BilinearForm`, where the `ext` got added. 
We can talk about it before making the changes.

Reply:
Decided to keep it like this.